Gemfile

```diff
@@ -15,9 +15,10 @@ gem 'virtus'
 gem 'rails-api'
 gem 'figaro'
 gem 'pg'
-
 gem 'active_model_serializers', '~> 0.10.0'
-
+gem 'will_paginate'
+gem 'responders'
```

Comment:
But you could accomplish the same thing with `render json: data, ...`
From what I gathered, the gem provides a way to respond to different formats without the `respond_to` block, but since we are always responding with JSON, it doesn't seem to provide much value